src/main/kotlin/com/hedvig/underwriter/model/InsuranceCompany.kt

```diff
@@ -0,0 +1,45 @@
+package com.hedvig.underwriter.model
+
+import com.neovisionaries.i18n.CountryCode
+
+data class InsuranceCompany(
+    val id: String,
+    val displayName: String,
+    val switchable: Boolean
+) {
+    companion object {
+
+        private val other = InsuranceCompany("other", "Other", false)
+
+        val insurersByCountry = mapOf(
```

Comment:
Let's not talk Country (it's an illegal word here) 😨 
```suggestion
        val insurersByCountryCode = mapOf(
```